src/nvoptix_68.c

```diff
@@ -0,0 +1,452 @@
+{
+    TRACE("(%p, %p, %u, %u, %p, %zu, %p, %zu)\n", denoiser, stream, inputWidth, inputHeight, state, stateSizeInBytes, scratch, scratchSizeInBytes);
+    return optixFunctionTable_68.optixDenoiserSetup(denoiser, stream, inputWidth, inputHeight, state, stateSizeInBytes, scratch, scratchSizeInBytes);
+}
+
+static OptixResult __cdecl optixDenoiserInvoke_68(OptixDenoiser denoiser, CUstream stream, const void *params, CUdeviceptr denoiserState, size_t denoiserStateSizeInBytes, const void *guideLayer, const void *layers, unsigned int numLayers, unsigned int inputOffsetX, unsigned int inputOffsetY, CUdeviceptr scratch, size_t scratchSizeInBytes)
+{
+    TRACE("(%p, %p, %p, %p, %zu, %p, %p, %u, %u, %u, %p, %zu)\n", denoiser, stream, params, denoiserState, denoiserStateSizeInBytes, guideLayer, layers, numLayers, inputOffsetX, inputOffsetY, scratch, scratchSizeInBytes);
+    return optixFunctionTable_68.optixDenoiserInvoke(denoiser, stream, params, denoiserState, denoiserStateSizeInBytes, guideLayer, layers, numLayers, inputOffsetX, inputOffsetY, scratch, scratchSizeInBytes);
+}
+
+static OptixResult __cdecl optixDenoiserComputeIntensity_68(OptixDenoiser handle, CUstream stream, const void *inputImage, CUdeviceptr outputIntensity, CUdeviceptr scratch, size_t scratchSizeInBytes)
+{
+    TRACE("(%p, %p, %p, %p, %p, %zu)\n", handle, stream, inputImage, outputIntensity, scratch, scratchSizeInBytes);
+    return optixFunctionTable_68.optixDenoiserComputeIntensity(handle, stream, inputImage, outputIntensity, scratch, scratchSizeInBytes);
+}
+
+static OptixResult __cdecl optixDenoiserComputeAverageColor_68(OptixDenoiser handle, CUstream stream, const void *inputImage, CUdeviceptr outputAverageColor, CUdeviceptr scratch, size_t scratchSizeInBytes)
+{
+    TRACE("(%p, %p, %p, %p, %p, %zu)\n", handle, stream, inputImage, outputAverageColor, scratch, scratchSizeInBytes);
+    return optixFunctionTable_68.optixDenoiserComputeAverageColor(handle, stream, inputImage, outputAverageColor, scratch, scratchSizeInBytes);
+}
+
+static OptixResult __cdecl optixDenoiserCreateWithUserModel_68(OptixDeviceContext context, const void *data, size_t dataSizeInBytes, OptixDenoiser *returnHandle)
+{
+    TRACE("(%p, %p, %zu, %p)\n", context, data, dataSizeInBytes, returnHandle);
+    return optixFunctionTable_68.optixDenoiserCreateWithUserModel(context, data, dataSizeInBytes, returnHandle);
+}
+
+OptixResult __cdecl optixQueryFunctionTable_68(
+    unsigned int numOptions,
+    int *optionKeys,
+    const void **optionValues,
+    void *functionTable,
+    size_t sizeOfTable)
+{
+    if (sizeOfTable != sizeof(OptixFunctionTable_68)) return OPTIX_ERROR_FUNCTION_TABLE_SIZE_MISMATCH;
+
+    OptixResult result = poptixQueryFunctionTable(60, numOptions, optionKeys, optionValues, &optixFunctionTable_68, sizeOfTable);
```

Comment:
```suggestion
    OptixResult result = poptixQueryFunctionTable(68, numOptions, optionKeys, optionValues, &optixFunctionTable_68, sizeOfTable);
```

Reply:
This was a critical miss by me tbh! Thanks for catching that one indeed 👍